lib/Service/ObjectService.php

```diff
@@ -67,7 +69,31 @@ public function __construct(
         $this->auditTrailMapper = $auditTrailMapper;
     }
 
-    /**
+	/**
+	 * Validate an object with a schema.
+	 * If schema is not given and schemaObject is filled, the object will validate to the schemaObject.
+	 *
+	 * @param array    $object		 The object to validate.
+	 * @param int|null $schema		 The id of the schema to validate to.
+	 * @param object   $schemaObject A schema object to validate to.
+	 *
+	 * @return ValidationResult The validation result from opis/json-schema.
+	 */
+	public function validateObject(array $object, ?int $schema = null, object $schemaObject = new stdClass()): ValidationResult
```

Comment:
I would prefer to rename $schema => $schemaId to make things a bit more clear

Reply:
I agree, thanks!